index.js

```diff
@@ -63,7 +63,7 @@ PublishRelease.prototype.publish = function publish () {
           'User-Agent': 'publish-release ' + pkg.version + ' (https://github.com/remixz/publish-release)'
         }
       }, function (err, res, body) {
-        if (err) return callback(err)
+        if (err || body.errors) return callback(err || body.errors)
```

Comment:
It's probably better to return error instances. `body.errors`, is that a string or an object?